docs/configuration.md

```diff
@@ -1638,7 +1638,7 @@ Apart from these, the following properties are also available, and may be useful
     For more detail, see the description
     <a href="job-scheduling.html#dynamic-resource-allocation">here</a>.
     <br><br>
-    This requires <code>spark.shuffle.service.enabled</code> to be set.
+    This requires <code>spark.shuffle.service.enabled</code> to be set true.
```

Comment:
You are right, but usually is not very sure. Can not let the user to guess, the document needs to be accurately described. In addition, the spark project, several other are clearly described, spark.shuffle.service.enabled set true.
![1](https://user-images.githubusercontent.com/26266482/28858102-e488331a-7780-11e7-90da-9390d1659f35.png)
![2](https://user-images.githubusercontent.com/26266482/28858105-ecfb276e-7780-11e7-9f7f-b0d5448dcb62.png)